pom.xml

```diff
@@ -47,6 +47,7 @@
     <version.org.slf4j>1.7.25</version.org.slf4j>
     <version.org.mvel>2.4.4.Final</version.org.mvel>
     <version.org.kie7>7.29.0.Final</version.org.kie7>
+    <version.org.kie7_snapshot>7.31.0-SNAPSHOT</version.org.kie7_snapshot>
```

Comment:
please don't use snapshots of v7 in kogito as it will make it dependent on v7 work and thus potentially make it unstable

Reply:
I agree, this is temporary. Correct @mariofusco ?